cmd/commands/common.go

```diff
@@ -265,6 +267,77 @@ func promptSummaryToUser(ctx context.Context, finalParameters map[string]string,
 	return false, nil
 }
 
+func getKubeContextNameFromUserSelect(cmd *cobra.Command, kubeContextName *string) error {
+	if !store.Get().Silent {
```

Comment:
you can switch the condition, return `nil`, and have the rest of the func not-nested.